var/spack/repos/builtin/packages/kafka/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Kafka(Package):
+    """
+    Kafka is used for building real-time data pipelines and streaming apps.
+    It is horizontally scalable, fault-tolerant, wicked fast, and runs in
+    production in thousands of companies.
+    """
+
+    homepage = "https://www-eu.apache.org/dist/kafka"
+    url      = "https://www-eu.apache.org/dist/kafka/2.3.1/kafka_2.12-2.3.1.tgz"
+    list_url = "https://www-eu.apache.org/dist/kafka/"
+    list_depth = 2
```

Comment:
Does this work when you set it to 1?

Reply:
Refine it in https://github.com/spack/spack/pull/14315/commits/3a5cd0d55ce36d444812810f17789f66a8fc4274